scipy/signal/fir_filter_design.py

```diff
@@ -583,3 +583,84 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
     bands = np.asarray(bands).copy()
     return sigtools._remez(numtaps, bands, desired, weight, tnum, Hz,
                            maxiter, grid_density)
+
+
+def firls(numtaps, bands, desired, weight=None, Hz=1):
+    """
+    FIR filter design using least-squares error minimization.
+
+    Calculate the filter coefficients for the finite impulse response
+    (FIR) filter which has the best approximation to the desired
+    frequency response described by `bands` and `desired` in the
+    least squares sense.
+
+    Parameters
+    ----------
+    numtaps : int
+        The number of taps in the FIR filter.  `numtaps` must be odd.
+    bands : array_like
+        A monotonic sequence containing the band edges in Hz.
+        All elements must be non-negative and less than half the sampling
+        frequency as given by `Hz`.
+    desired : array_like
+        A sequence half the size of bands containing the desired gain
+        in each of the specified bands.
+    weight : array_like, optional
+        A relative weighting to give to each band region. The length of
+        `weight` has to be half the length of `bands`.
+    Hz : scalar, optional
+        The sampling frequency in Hz. Default is 1.
```

Comment:
This should not say `in Hz`.  There is no requirement that the units of the sampling rate be Hz.